website/docs/r/security_center_subscription_pricing.html.markdown

```diff
@@ -21,13 +21,36 @@ resource "azurerm_security_center_subscription_pricing" "example" {
 }
 ```
 
+Example for extensions
+```hcl
+resource "azurerm_security_center_subscription_pricing" "test" {
+  tier= "Standard"
+  resource_type = "CloudPosture"
+  extension {
+    name= "SensitiveDataDiscovery"
+    enabled = "True"
+  }
+  extension {
+    name= "AgentlessDiscoveryForKubernetes"
+    enabled = "False"
+  }
+  extension {
+    name= "AgentlessVmScanning"
+    enabled = "True"
+    additional_extension_properties = {
+        ExclusionTags = "[]"
+    }
+  }
+}
+```
```

Comment:
generally we only include properties that are "required" for the resource to work in the example, so we can probably remove this

Reply:
Sure